scripts/purge_mem_test.py

```diff
@@ -0,0 +1,33 @@
+import time
+
+import smart_open
+from HTC import HT_Basic as HashTable
+
+
+def loadWiki(counter, articles=1):
```

Comment:
For function name please don't use `camelCase`, only lowercase.

Reply:
`lowercase_with_underscore` in Python.